paimon-core/src/main/java/org/apache/paimon/catalog/LockContextUtils.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.paimon.catalog;
+
+import org.apache.paimon.jdbc.JdbcCatalogFactory;
+import org.apache.paimon.jdbc.JdbcCatalogLock;
+import org.apache.paimon.jdbc.JdbcClientPool;
+import org.apache.paimon.jdbc.JdbcUtils;
+import org.apache.paimon.options.CatalogOptions;
+import org.apache.paimon.options.Options;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.SQLException;
+import java.util.Optional;
+
+/** Utils for {@link org.apache.paimon.catalog.CatalogLock.LockContext}. */
+public class LockContextUtils {
+
+    private static final Logger LOG = LoggerFactory.getLogger(FileSystemCatalog.class);
+
+    private static JdbcClientPool connections;
```

Comment:
Why use static member here?

Reply:
> Why using static member here?

@JingsongLi The main reason is that I declared all the methods in utils as static, and this place will not be destroyed immediately after use. It can be destroyed together with the process release, so I declared it as static

Reply:
I mean why it should be destroyed together with the process release.